src/main/java/com/meta/cp4m/message/FBMessageHandler.java

```diff
@@ -44,6 +44,7 @@ public class FBMessageHandler implements MessageHandler<FBMessage> {
   private final String appSecret;
 
   private final String accessToken;
+  @Nullable private final String connectedFacebookPageForInstagram;
```

Comment:
```suggestion
  private final @Nullable String connectedFacebookPageForInstagram;
```
This just keeps the reformatting from adding a newline